### PR TITLE
Docs update for 1.10.0 release

### PR DESCRIPTION
**Pull request by michalbiesek**
### Types of changes

- [ ] Bugfix (non-breaking change which fixes issue linked in Description above)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (correction or otherwise)
- [ ] Cosmetics (whitespace, appearance)
- [ ] Other

### Checklist

- [ ] Code compiles without errors
- [ ] All tests pass locally with my changes (see TESTING section in CONTRIBUTING file)
- [ ] Created tests which will fail without the change (if possible)
- [ ] Extended the README/documentation (if necessary)
- [ ] All newly added files have proprietary license (if necessary)
- [ ] All newly added files are referenced in MANIFEST files (if necessary)

## Further comments

---
This change is [Reviewable](https://reviewable.io/reviews/memkind/memkind/323)

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,13 @@
+* Mon Dec 16 2019 Michal Biesek <michal.biesek@intel.com> v1.10.0
+- Provided second way to use Persistent Memory in memkind (MEMKIND_DAX_KMEM_* kinds)
+- Added C++ allocator for static kinds
+- Provided interface to get memkind allocation statistics
+- Provided method to reduce fragmentation - memkind_defrag_reallocate()
+- Added suppport of background thread
```
Comment (jschmieg) [inline ChangeLog]: support

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+- Updated internally used jemalloc to upstream version 5.2.1
+- Extended hbw interface with hbw_malloc_usable_size()
+- Simplify buliding process
```
Comment (jschmieg) [inline ChangeLog]: building

File: ChangeLog (hunk 1 above)
Comment (michalbiesek) [reply ChangeLog]: Done.

File: ChangeLog (hunk 2 above)
Comment (michalbiesek) [reply ChangeLog]: Done.

**Pull request merged**